File: hotel/latihan_hotel/3.py
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
import re

df = pd.read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-02-11/hotels.csv')
city = df[df['hotel'] == 'City Hotel']
hasil = city['is_canceled'].value_counts(normalize=True)
print('City Hotel')
print(hasil)
print()
resort = df[df['hotel'] == 'Resort Hotel']
hasil = resort['is_canceled'].value_counts(normalize=True)
print('Resort Hotel')
print(hasil)

File: hotel/latihan_hotel/3.r
library(dplyr)
library(tidyverse)
library(ggplot2)

df_hotel <- read.csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-02-11/hotels.csv', na.strings = c("", "NA", " ", "N/A", "NULL"))
# Assuming df_hotel is your data frame
df_hotel <- df_hotel %>% mutate(ID = row_number()) %>% select(ID, everything())
city <- filter(df_hotel, hotel == 'City Hotel')
hasil <- prop.table(table(city$is_canceled))
print(hasil)


resort <- filter(df_hotel, hotel == 'Resort Hotel')
hasil <- prop.table(table(resort$is_canceled))
print(hasil)